File: ApiSuvesaPos/Negocio/Modelo/inventario/detalle_empaquetado.vb
Namespace Modelo
    Public Class detalle_empaquetado
        'modelo usado por sistema de entregas
        Public Property id_venta_detalle As Integer
        Public Property Codarticulo As String
        Public Property Descripcion As String
        Public Property Cantidad As Decimal
        Public Property minutos As String
        Public Property Selección As Boolean
        Public Property codmarca As String
        Public Property Id_Factura As Integer
        Public Property Num_Factura As Integer
        Public Property devoluciones As Decimal
        Public Property anulado As Boolean
        Public Property nota_pantalla As String
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/inventario/detalle_empaquetado.cs
namespace Modelo
{
    public class detalle_empaquetado
    {
        // modelo usado por sistema de entregas
        public int id_venta_detalle { get; set; }
        public string Codarticulo { get; set; }
        public string Descripcion { get; set; }
        public decimal Cantidad { get; set; }
        public string minutos { get; set; }
        public bool Selección { get; set; }
        public string codmarca { get; set; }
        public int Id_Factura { get; set; }
        public int Num_Factura { get; set; }
        public decimal devoluciones { get; set; }
        public bool anulado { get; set; }
        public string nota_pantalla { get; set; }
    }
}
